tests/examples/advanced_search.feature.example

```diff
@@ -0,0 +1,35 @@
+Feature: Advanced Search
+
+  Scenario: date range search
+    Dates can be searched across all versions and the results show the most current version.
+    Not sure how to write the test to reflect this.
+
+    Given Specific year is "2015"
+    When a user performs a search
+    Then a list of results is returned
+    And all entries contain a metadata date field that matches "2015"
```

Comment:
Would we specifically expect that the submitted date would be in the year 2015?

Reply:
The intent was some logic phrase that says "any date field associated with this entry that contains 2015" - but you're right, the way we discussed for it to behave is along the lines of "the submitted date field for any of the versions associated with this entry contains 2015". (Right?)

Reply:
Yeah, that sounds right to me